### PR TITLE
Search Teeing extension side [WIP]

### PR DESCRIPTION
**Pull request by stkem**
50% of searches go through shoebox (which T's them to two search machines and returns the fastest),
so we can measure the performance difference with some real traffic.
Also keeps the api.kifi.com dns and ssl session warm.
RFC @raymondng 
cc @eishay @ymatsuda @2is10

### CODE REVIEW

File: packrat/main.js [hunk 1]
```diff
@@ -160,6 +162,13 @@ function insertUpdateChronologically(arr, o, time) {
 
 // ===== Server requests
 
+function keepApiWarm() {
+  var now = new Date().getTime()
```
Comment (2is10) [inline packrat/main.js]: `Date.now()` is equivalent and cheaper

File: packrat/main.js [hunk 2]
```diff
@@ -160,6 +162,13 @@ function insertUpdateChronologically(arr, o, time) {
...
+  if (!lastApiCall || lastApiCall<(now-20000)) {
```
Comment (2is10) [inline packrat/main.js]: We use spaces around operators like `<` and `-` and `===`. Thanks.

File: packrat/main.js [hunk 3]
```diff
@@ -1704,6 +1722,11 @@ function startSession(callback, retryMs) {
     logEvent("extension", "authenticated");
     unstore('logout');
 
+    keepApiWarm();
+    setInterval(function(){
+      keepApiWarm();
+    }, 5000);
```
Comment (2is10) [inline packrat/main.js]: Let’s remove this and the `lastApiCall` tracking for now. I’ll address the DNS concern later today. Thanks.

**Comment by stkem** [comment]
@2is10 ok to merge in it's current state?

**Pull request merged**